packages/maskbook/src/web3/UI/TxFeeEstimation.tsx

```diff
@@ -0,0 +1,79 @@
+import { Grid, Paper, Typography, makeStyles, useTheme } from '@material-ui/core'
+import { useI18N, useRemoteControlledDialog } from '../../utils'
+import { Image } from '../../components/shared/Image'
+import { useAssets } from '../../plugins/Wallet/hooks/useAssets'
+import { EthereumMessages } from '../../plugins/Ethereum/messages'
+import { useStylesExtends } from '../../components/custom-ui-helper'
+import { useValueRef, formatWeiToGwei, formatBalance } from '@dimensiondev/maskbook-shared'
+import { GasNow, useGasPrice, useNetworkType, NetworkType, EthereumTokenType } from '@dimensiondev/web3-shared'
+import { currentGasNowSettings } from '../../plugins/Wallet/settings'
+import { useState, useCallback, useMemo } from 'react'
+
+const useStyles = makeStyles(() => {})
+
+interface TxFeeEstimationProps extends withClasses<'label' | 'button' | 'data' | 'gasEstimation'> {
+    gas?: number
+}
+const TUNE_ICON_DARK_MODE = new URL('../assets/tune_dark_mode.png', import.meta.url).toString()
```

Comment:
Why not to use MUI icons?

Reply:
MUI icons is not as same as our design on figma.